res/css/focus.css

```diff
@@ -0,0 +1,17 @@
+/* the box shadow is taken from photon-button primary button focus styling
+https://design.firefox.com/photon/components/buttons.html */
+
+/* removes the default Firefox focus indicator,
+so that we can add a custom indicator */
+input::-moz-focus-inner,
+button::-moz-focus-inner {
+  border: 0;
+}
+
+a:focus,
+input:focus,
```

Comment:
there's a problem here, in that this should apply only to some inputs but not all. Especially this should apply to `input[type=radio]` `checkbox` `button` `submit` `reset` `image` `range`.
I think this should also apply to `select` actually.

Sorry, I should have seen this before... This is especially visible for the search input at the right of the window: with your current styles it has a double border or something like this.

Other inputs, as well as `textarea`, should get the focus style that I already implemented in https://github.com/devtools-html/perf.html/blob/6ca6e638fda3c3ccfab244c70eeabab27a8296bc/res/css/photon-components.css#L88-L89. But maybe it's ok to not implement this part here and rather decide that all such inputs (eg: the search input) should use the class `.photon-input`? 

(I think this class wasn't existing yet when you started working on this topic)